src/translators/crate.py

```diff
@@ -56,7 +56,8 @@ def setup(self):
         self.connection = self.ccm.get_connection('crate')
         if self.connection is None:
             try:
-                self.connection = client.connect([url], error_trace=True)
+                # Added backoff_factor for retry interval between attempt of consecutive retries
+                self.connection = client.connect([url], error_trace=True, backoff_factor=0.1)
```

Comment:
@c0c0n3 have a look here, i am not sure what's the impact on queue workers.
also shouldn't this be configurable?

Reply:
> @c0c0n3 have a look here, i am not sure what's the impact on queue workers.
> also shouldn't this be configurable?

@chicco785 , i can add `backoff_factor` as environment variable. If it's ok with you then i can modify code?

Reply:
> have a look here, i am not sure what's the impact on queue workers.

It should be okay, but there must be a way to configure it since w/ WQ you also have retry intervals, so the admin should be able to configure the Crate client's backoff factor to make sure the total amount spent by the client retrying a connection is less than the minimum retry interval in WQ...

> i can add backoff_factor as environment variable.

yes please!

Reply:
@c0c0n3 is there away to check this?

Reply:
i meant more an automatic check at start that verify that the back off is smaller than retry interval (which i am not sure how it is set)